exasol_transformers_extension/deployment/scripts_deployer.py

```diff
@@ -48,6 +49,14 @@ def deploy_scripts(self) -> None:
     def run(cls, dsn: str, user: str, password: str,
             schema: str, language_alias: str):
 
-        pyexasol_conn = pyexasol.connect(dsn=dsn, user=user, password=password)
+        pyexasol_conn = pyexasol.connect(
+            dsn=dsn,
+            user=user,
+            password=password,
+            encryption=True,
+            websocket_sslopt={
+                "cert_reqs": ssl.CERT_NONE,
```

Comment:
For every functionality which is not test related, I would recommend enable the user of the functionality to decide on enabling/disabling the certificate check.